core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1921,9 +1921,6 @@ secret_configs:
   - cluster: app.ci
     name: prometheus-auth-credentials
     namespace: prow-monitoring
-  - cluster: app.ci
```

Comment:
Err I think this and everything below needs to stay, why did you remove it?

Reply:
what the hell...